CHANGELOG_iOS.md

```diff
@@ -1,5 +1,33 @@
 # Changelog
 
+## [1.81.133](https://github.com/brave/brave-browser/releases/tag/v1.81.133)
+
+### Web3 
+
+ - Added support for resolving new Unstoppable Domains TLDs including .brave TLD. ([#46271](https://github.com/brave/brave-browser/issues/46271))
+ - Added a Brave proxy for Zcash mainnet endpoint. ([#46266](https://github.com/brave/brave-browser/issues/46266))
+ - Enabled ZCash feature flag by default. ([#48171](https://github.com/brave/brave-browser/issues/48171))
+ - Fixed issue for web views not being reloaded when connected to https://jup.ag DApp after changing default Solana wallet preference. ([#47676](https://github.com/brave/brave-browser/issues/47676))
```

Comment:
no need to specify the Dapp. it was a general issue not specific to that site. so can rename to 

`Fixed issue with web-views not reloading when detault Solana Wallet  provider is changed`